core/include/macros/xtstr.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2019 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     core_internal
+ * @{
+ *
+ * @file
+ * @brief       Common macros
```

Comment:
I guess this description should change now?

Reply:
I added them to a different group as well let me know if that makes sense

Reply:
Thanks!